docs/source/release/v5.0.0/mantidworkbench.rst

```diff
@@ -26,94 +33,127 @@ New
+   *Toggle Error Notification*
+
+- All the relevant settings from manitdplot have been added to workbench.
+- Most changes in the settings dialog now take place immediately, no longer needing a restart, such as changing default instrument, hiding categories of algorithms and interfaces, or choosing whether to see invisible workspaces.
+
+**Fitting**
 - Fit functions can now be put into nested categories and into multiple categories.
+- Now able to exclude regions of data from a fit, within the fit property browser.
+- You can now search for functions when doing fits.
+- A help button has been added to the fitting add function dialog.
+
+**Plotting**
+- Figure options>Images tab no longer requires Max > Min value. If Min > Max, they will be swapped!
+- A warning now appears if you attempt to plot more than ten spectra.
 - Normalization options have been added to 2d plots and sliceviewer.
-- An exclude property has been added to the fit property browser
-- The images tab in figure options no longer forces the max value to be greater than the min value.
-- The algorithm progress details dialog now fills immediately with all running algorithms rather than waiting for a progress update for the algorithm to appear.
-- All the relevant settings from manitdplot have been added to workbench
 - Double clicking on a workspace that only has a single bin of data (for example from a constant wavelength source) will now plot that bin, also for single bin workspaces a plot bin option has been added to the right click plot menu of the workspace.
-- Default values for algorithm properties now appear as placeholder (greyed-out) text on custm algorithm dialogs.
 - The context menu for WorkspaceGroups now contains plotting options so you can plot all of the workspaces in the group.
-- Most changes in the settings dialog now take place immediately, no longer needing a restart, such as hiding algorithm categories, interfaces or choosing wether to see invisible workspaces.
-- A warning now appears if you attempt to plot more than ten spectra.
+
+.. figure:: ../../images/SaveButton.png
+   :align: right
+
+   *Save drop-down menu*
+
+**Algorithms**
+- Table Workspaces can be saved to Ascii using the :ref:`SaveAscii <algm-SaveAscii>` algorithm, and the *Save > Ascii drop-down* option above the Workspaces Toolbox.
+- The *Save drop-down* menu in the workspaces toolbox no longer has the old SaveAscii (version 1) as no one was using it and it only added confusion. The most recent version of SaveASCII is still available.
+- The algorithm progress details dialog now fills immediately with all running algorithms rather than waiting for a progress update for the algorithm to appear.
+- Algorithm dialogs will now use the selected workspace as the InputWorkspace when running an algorithm from the algorithms toolbox.
+
+.. figure:: ../../images/SaveButton.png
```

Comment:
I think this is supposed to be a different image? Currently it's the same as the one above.